src/pages/messages/components/DeleteIconButton.tsx

```diff
@@ -0,0 +1,30 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { IconButton, IconButtonProps, Tooltip } from '@mui/material';
+import { mainTheme } from '../../../themes/mainTheme';
+
+interface Props extends IconButtonProps {
+  tooltip?: string;
+}
+
+export const DeleteIconButton = ({ sx, tooltip, ...rest }: Props) => (
+  <Tooltip title={tooltip}>
+    <IconButton
+      sx={{
+        bgcolor: mainTheme.palette.primary.main,
+        '&:hover': {
+          bgcolor: mainTheme.palette.primary.main,
```

Comment:
kanskje litt annen farge for å indikere hover? Ser at `main` og `dark` er veeeeldig like der da men..
```suggestion
          bgcolor: 'primary.dark',
```

Reply:
Ja det er bra! Jeg slet litt med å finne ut hva jeg skulle gjøre der